views/gitHub-loading.hbs

```diff
@@ -0,0 +1,51 @@
+<html>
```

Comment:
We should probably include the name `session` in the name of this file e.g. gitHub-server-session. Also, it probably makes sense to rename the current `session.hbs` file to something like gitHub-cloud-session.

Reply:
Oh, I just realised you're rendering both cloud and server in this file. We can probably just delete the session.hbs file and rename this to just session.hbs or github-session.hbs

Reply:
`session.hbs` has been deleted and this file has been renamed to `gitHub-session.hbs`.